vivisect/tests/teststorage.py

```diff
@@ -114,3 +114,13 @@ def test_msgpack_to_basicfile(self):
             basicfile.close()
             os.unlink(mpfile.name)
             os.unlink(basicfile.name)
+
+    def test_bad_event(self):
+        vw = vivisect.VivWorkspace()
+        with self.assertLogs() as logcap:
+            vw.importWorkspace([(VWE_MAX + 1, (0xabad1dea, 4, 3, 'nope')),
+                                (VWE_ADDFILE, ('VivisectFile', 0x1000, '3bfdad02b9a6522c84e356cf8f69135b'))])
+        files = vw.getFiles()
+        self.assertIn("IndexError: list index out of range", ''.join(logcap.output))
```

Comment:
does this work?  and is it bad form to have this outside the `logcap` scope?

Reply:
Yup! It's how the python documentation models using it, so I figure it should work.

(relevant: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs)

Reply:
wow, go fig.